src/python.gen.ts

```diff
@@ -464,7 +464,7 @@ ${this.hooks.join('\n')}
       if (pipEnvExists.includes('pipenv')) {
         // pipenv check completed without error
         run('pipenv', ['update'])
-        run('pipenv', ['run', 'black', `${this.codePath}/${this.packagePath}/sdk/`])
+        run('pipenv', ['run', 'black', `${this.codePath}/${this.packagePath}/sdk/*.py`])
```

Comment:
Is this called only once for all files? It appears to be called for every file.

Reply:
I think `black` is only called once and the `*.py` in the path argument is needed to tell it what files to run on. I'd thought it only needed a directory but maybe that changed because it wasn't formatting files. Here's the output:

```
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
⠇ Locking..✔ Success!
Updated Pipfile.lock (595abd)!
reformatted /Users/joeldodge/Documents/sdk_codegen/python/looker_sdk/sdk/methods.py
reformatted /Users/joeldodge/Documents/sdk_codegen/python/looker_sdk/sdk/models.py
All done! ✨ 🍰 ✨
2 files reformatted, 1 file left unchanged.
```

(the `1 file left unchanged` is `python/looker_sdk/sdk/__init__.py` which is empty)

Reply:
`reformatFile()` is called for each source file, currently. We could refactor for the Python scenario of a folder.

```typescript
// Reformat source files after generation
  reformat() {
    const result: string[] = []
    let files = ['sdk/methods', 'sdk/models']
    if (this.willItStream) files.push('sdk/streams')
    for (const name of files) {
      const sourceFile = this.fileName(name)
      const output = this.reformatFile(sourceFile)
      if (output) {
        result.push(output)
      }
    }
    return result
  }
```

Reply:
oh geez, I totally missed that. I'll update